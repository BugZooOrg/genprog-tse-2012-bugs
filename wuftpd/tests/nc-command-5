user invalid-user
pass invalid-password
site exec /a/b
quit
