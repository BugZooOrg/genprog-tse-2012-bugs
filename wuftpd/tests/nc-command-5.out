331 Password required for invalid-user.
530 Login incorrect.
530 Please login with USER and PASS.
221 Goodbye.
